.github/workflows/cd.yml

```diff
@@ -16,6 +16,7 @@ jobs:
       TARGET_PROJECT_KEY: java-sync-target-dev2
       TARGET_CLIENT_ID: ${{ secrets.TARGET_CLIENT_ID_2 }}
       TARGET_CLIENT_SECRET: ${{ secrets.TARGET_CLIENT_SECRET_2 }}
+      CI_BUILD_DIR: '/home/runner/work/commercetools-sync-java/commercetools-sync-java/'
```

Comment:
The changes coming with the other commits should not be overridden, could you please check and merge the commits without conflicts, see: https://github.com/commercetools/commercetools-sync-java/commit/c3e039947bc13f68ab31f317fdeb6fbbff2bd937

Reply:
3ff6579